### PR TITLE
Resolved CA 2213: Closed a disposable field

### PR DESCRIPTION
**Pull request by jericahuang**
Closed the disposable field _jsonTextWriter

@lgolding @michaelcfanning @Rolstenhouse @kschecht

### CODE REVIEW

File: src/Sarif/Writers/SarifLogger.cs [hunk 1]
```diff
@@ -198,7 +198,7 @@ public void Dispose()
                     _issueLogJsonWriter.WriteToolNotifications(_run.ToolNotifications);
                 }
 
-                if (_run != null && 
+                if (_run != null &&
```
Comment (jericahuang) [inline src/Sarif/Writers/SarifLogger.cs]: I didn't change this line... don't know why it's highlighted in the diff.

**Comment by jericahuang** [comment]
Never mind, this raised an error. Closing for now.

**Comment by jericahuang** [comment]
Error's gone, all tests pass.

File: src/Sarif/Writers/SarifLogger.cs (hunk 1 above)
Comment (lgolding) [inline src/Sarif/Writers/SarifLogger.cs]: If you look carefully at the left side, you'll see that a single space at the end of the line has been removed (which is good :-)).

**Comment by lgolding** [comment]
@jericahuang Go ahead and merge. @michaelcfanning wants to see what work is being done but I know he'll approve this change.

**Pull request merged**